src/libraries/Microsoft.PowerFx.Core/Texl/Intellisense/IntellisenseResult.cs

```diff
@@ -144,14 +147,15 @@ internal IntellisenseResult(IIntellisenseData data, List<IntellisenseSuggestion>
                         funcDisplayString.Append(')');
                         var signatureInformation = new SignatureInformation()
                         {
-                            Documentation = func.Description,
+                            // Use overload description and not IntellisenseData.CurrFunc overload
+                            Documentation = possibleOverload.Description ?? func.Description,
```

Comment:
>ossibleOverload.Description ?? func.Description, [](http://example.com/codeflow?start=45&length=48)

Can we also add func.Description when the Description is empty string?

Reply:
Not sure if i understand. Am i not doing that already? If possibleOverload.Description is empty then this would fallback to func.Description.

Reply:
Oh oops never mind. this is null collasceing operator so it won't cover empty string

Reply:
done